kernal/cbm/editor.s

```diff
@@ -230,7 +230,7 @@ stok	jsr screen_set_position
 ;
 	lda llen
 .ifp02
-	clc
+	clc ; XXX is this correct?? (not reached because X16 is 65C02)
```

Comment:
Neither do I understand that. I shouldn't be equal to the DEC instruction used when compiling for the 65C02.

Reply:
I checked the C64 disassembly. `llen` is a constant and it does `lda #llen-1`
This definitely looks like a bug, but I don't know if this code will ever get used on a non-C02 system.

I think I may push a commit that changes these two `clc`s to `sec`